### PR TITLE
Project heatlh domain threshold

### PR DESCRIPTION
**Pull request by czue**
this adds a setting to internal domain management that allows projects to define their own "performing" threshold in the project performance report.

### CODE REVIEW

File: corehq/apps/domain/models.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ class InternalProperties(DocumentSchema, UpdatableSchema):
     goal_followup_rate = DecimalProperty()
     # intentionally different from and commtrack_enabled so that FMs can change
     commtrack_domain = BooleanProperty()
+    performance_threshold = IntegerProperty()
```
Comment (benrudolph) [inline corehq/apps/domain/models.py]: nit: this could be `default=15`

**Comment by benrudolph** [comment]
seems good to me. 👍 on ✔️

File: corehq/apps/domain/models.py (hunk 1 above)
Comment (czue) [inline corehq/apps/domain/models.py]: i actually considered that and decided against since it would be much harder to change the default later

Comment (benrudolph) [inline corehq/apps/domain/models.py]: 👍

**Pull request merged**